package.json

```diff
@@ -1,29 +1,74 @@
 {
-  "name": "revelio",
-  "version": "1.0.0",
+  "name": "@connexta/revelio",
+  "version": "0.0.1",
   "main": "src/main/webapp/main.js",
   "license": "MIT",
   "devDependencies": {
-    "@connexta/ace": "https://github.com/connexta/ace.git#81e7b8b22b3f7e43b356ba5dfbc065ffdb694892",
+    "@apollo/react-hooks": "^3.1.3",
```

Comment:
Moved all dependencies to dev and peer dependencies so downstream project has to have the dependency. Maybe this is a bad idea?

Reply:
With these in `dependencies` the yarn.lock when importing looks like this:
```
+a-test-of-revelio@1.0.47:
+  version "1.0.47"
+  resolved "https://registry.yarnpkg.com/a-test-of-revelio/-/a-test-of-revelio-1.0.47.tgz#de3902a70813342fd9fe0f1434bc243bfd3f2362"
+  integrity sha512-MGhFVegnDCzTMp15AFL/vFSL/FiZnIqqDzrcF5pYsVeHqo/3E0az4Hl+JxZD8MyOWxQrkcSzg5al+cMOAgya0Q==
+  dependencies:
+    "@apollo/react-hooks" "^3.1.3"
+    "@apollo/react-ssr" "^3.1.3"
+    "@date-io/moment" "1.3.13"
+    "@material-ui/core" "^4.6.1"
+    "@material-ui/icons" "^4.2.1"
+    "@material-ui/lab" "^4.0.0-alpha.32"
+    "@material-ui/pickers" "^3.2.5"
+    a-test-of-revelio "^1.0.40"
+    abort-controller "^3.0.0"
+    apollo-cache-inmemory "^1.6.3"
+    apollo-client "^2.6.4"
+    apollo-link "^1.2.13"
+    apollo-link-batch-http "^1.2.13"
+    apollo-link-error "^1.1.12"
+    apollo-link-http "^1.5.16"
+    apollo-link-schema "^1.2.4"
+    apollo-server-express "^2.9.7"
+    compression "^1.7.4"
+    express "4.17.1"
+    geospatialdraw "0.5.5"
+    golden-layout "^1.5.9"
+    graphql "^14.5.8"
+    graphql-tag "^2.10.1"
+    graphql-tools "^4.0.5"
+    immutable "^4.0.0-rc.12"
+    lodash.throttle "^4.1.1"
+    moment "^2.24.0"
+    moment-timezone "^0.5.16"
+    ol "^6.0.1"
+    on-finished "^2.3.0"
+    polished "^3.4.1"
+    qs "^6.9.1"
+    react "^16.9.0"
+    react-dom "^16.9.0"
+    react-loadable "^5.5.0"
+    react-redux "^7.1.0"
+    react-router "^5.1.2"
+    react-router-dom "^5.1.2"
+    redux "^4.0.4"
+    redux-thunk "^2.3.0"
+    styled-components "^4.3.2"
+    universal-cookie "^4.0.2"
+    uuid "^3.3.3"
+    winston "^3.2.1"

+a-test-of-revelio@^1.0.40:
+  version "1.0.48"
+  resolved "https://registry.yarnpkg.com/a-test-of-revelio/-/a-test-of-revelio-1.0.48.tgz#178d3fd46b130288749329dd8fcca594da3793e6"
+  integrity sha512-08Tme3qmHN2XxERoL0IImio26ms0NwVWm5F97HNVsmSMorpQPz/cAf0uq/4ns6DyCN7zqfQOVVSNz8nOm0/Aow==
```

With them moved to `peer/devDependencies`:
```
 
+a-test-of-revelio@^1.0.49:
+  version "1.0.49"
+  resolved "https://registry.yarnpkg.com/a-test-of-revelio/-/a-test-of-revelio-1.0.49.tgz#054eb58e7758ba33c3ed734e041b2417d3d99360"
+  integrity sha512-3i5B+bIhTwClSyCs9NZs9bkgexIjlWhPnzr84hz4oyMFmBabaCXng5G7IzIOpbZ/Tvo+tRDdR8n6oMJo2Nhc1w==

```